lib/mongoDB.js

```diff
@@ -11,7 +13,7 @@ const dbName = encodeURIComponent(Env.MongoDB);
 
 let _mongoConnection = null;
 let connection = null;
-let url = "mongodb://" + user + ":" + password + "@" + host;
+let url = "mongodb+srv://" + user + ":" + password + "@" + host + '/' + dbName + '?retryWrites=true&w=majority';
```

Comment:
What's w majority ?

Reply:
write majority:
most of what you insert will get written. But not a 100%